napari/viewer.py

```diff
@@ -142,6 +145,38 @@ def export_figure(
             flash=flash,
         )
 
+    def export_rois(
+        self,
+        rois: np.ndarray,
+        paths: list[str] | None = None,
+        scale: float | None = None,
+    ):
+        """Export the shapes rois with storage file paths
+
+        Parameters
+        ----------
+        rois: numpy array
+            An array of shape (n, 2, 2) where n is the number of rois
+            and the first two coordinates correspond to the top left and
+            the last two coordinates correspond to the bottom right corners
+        paths: list
+            The list to store file path for shapes roi
+
+        Returns
+        -------
+        roi_dict: dictionary
+            The dictionary with index and file paths for each shapes roi
```

Comment:
outdated docstring?

Reply:
leftover from when we were using shapes as inputs instead of numpy array.